src/clojurians_log/views.clj

```diff
@@ -247,10 +247,15 @@
      [:table
       [:tr
        [:td
-        [:div.app-title [:a {:href "/"} (get-in context [:request :clojurians-log.application/title])]]]
+        [:div.app-title
+         [:a {:href "/"}
+          (get-in context [:request :clojurians-log.application/title])]]]
        [:td.padding-15px
         [:a {:href (path-for context :clojurians-log.routes/about)}
-         "About"]]]]
+         "About"]]
+       [:td.padding-15px
+        [:a {:href (path-for context :clojurians-log.routes/sitemap)}
+         "Sitemap"]]]]
```

Comment:
Do we need this link to the sitemap, or just have it in robots.txt?

Reply:
Doesn't hurt to have it, but yes it also should go into robots.txt, which I guess we don't check in here.

Reply:
So I leave the link here. I amended the file resources/public/robots.txt as well.